spring-cloud-gcp-autoconfigure/src/main/resources/com/google/cloud/spring/autoconfigure/logging/logback-appender.xml

```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<!-- DEPRECATED!!! PLEASE USE "com/google/cloud/spring/logging/logback-appender.xml" -->
```

Comment:
I thought you already removed this file in another PR.

Reply:
See first commit message - I missed these two files.